content/6_Advanced/Wavelet.mdx

```diff
@@ -1,19 +1,17 @@
 ---
 id: wavelet
 title: 'Wavelet Tree'
-author: Benjamin Qi
+author: Benjamin Qi, Omri Jerbi
 prerequisites:
   - RURQ
-description: "?"
+description: Wavelet trees are data structures that support efficient queries for the k-th minimum element in a range by maintaining a segment tree over values instead of indices"
```

Comment:
keep the description brief, maybe like:

`Wavelet trees support efficient queries for the kth minimum element in a range`